jQuery/domManipulation.js

```diff
@@ -0,0 +1,65 @@
+window.honey = {};
```

Comment:
It almost doesn't matter. it could even be in a `script` tag before other scripts are loaded. 

e.g.

`<script> window.honey = {} </script>`